plugins/Admin/plugin.py

```diff
@@ -359,6 +359,20 @@ def acmd(self, irc, msg, args, commandAndArgs):
             self.Proxy(irc.getRealIrc(), msg, commandAndArgs)
     acmd = wrap(acmd, ['admin', many('something')])
 
+    class sasl(callbacks.Commands):
+
+        @internationalizeDocstring
+        def start(self, irc, msg, args):
```

Comment:
Move it to the Network plugin

Reply:
I would consider renaming this to something like `sasl auth` or `sasl login`. `start` is less descriptive and sounds like something that starts a longer running task or service, which isn't the case here.